core/src/main/scala/epollcat/unsafe/EpollExecutorScheduler.scala

```diff
@@ -43,7 +42,7 @@ private[unsafe] final class EpollExecutorScheduler private (
     val noCallbacks = callbacks.isEmpty()
 
     if ((timeoutIsInfinite || timeout == Duration.Zero) && noCallbacks)
-      false // nothing to do here
+      false // nothing to do here. refer to scaladoc on PollingExecutorScheduler#poll
```

Comment:
Thank you. Given the "refer to", even I could probably find that
(and will chase it down for my own edification).